src/scripts/demo.js

```
@@ -0,0 +1,12 @@
+import { createTree } from './main.js';
+
+const food = {
+  Drink: { Wine: {}, Schnaps: {} },
+  Fruit: {
+    Red: { Cherry: {}, Strawberry: {} },
+    Yellow: { Banana: {}, Pineapple: {} },
+  },
+};
+
+const tree = document.querySelector('#tree');
```

Potential timing issue: selecting `#tree` at module evaluation time can return `null` if the DOM element isn't present yet. `createTree` already guards against a falsy `element`, so this is safe, but the demo may silently do nothing if run too early. Consider wrapping the selection and call in a `DOMContentLoaded` listener or ensuring this script is loaded after the DOM.